src/ol_dbt/models/staging/ocw/stg__ocw__studio__postgres__websites_websitecontent.sql

```diff
@@ -42,6 +42,17 @@ with source as (
         , {{ cast_timestamp_to_iso8601('deleted') }} as websitecontent_deleted_on
         , {{ cast_timestamp_to_iso8601('created_on') }} as websitecontent_created_on
         , {{ cast_timestamp_to_iso8601('updated_on') }} as websitecontent_updated_on
+        -- miscellaneous metadata
+        , json_query(metadata, 'lax $.body' omit quotes) as metadata_body
+        , json_query(metadata, 'lax $.description' omit quotes) as metadata_description
+        , cast(json_query(metadata, 'lax $.draft' omit quotes) as boolean) as metadata_draft
```

Comment:
This line caused the casting error. There are '' as draft, so we could do something like this to fix it. But then we need to remove the not_null check on https://github.com/mitodl/ol-data-platform/pull/1158/files#diff-c31c92d68f6fc17a536484487b15af33c96544b8143ca75d91eaf36f22903c06R199
```suggestion
          , case
            when json_query(metadata, 'lax $.draft' omit quotes) = '' then null
            else cast(json_query(metadata, 'lax $.draft' omit quotes) as boolean)
        end as metadata_draft
```